app/views/bookmarks/_index_header_default.html.erb

```diff
@@ -1,5 +1,6 @@
 <%# header bar for doc items in index view -%>
-<%# overrides catalog/_index_header_default.html.erb to link to documents within the context of their exhibit%>
+<%# overrides blacklight's catalog/_index_header_default.html.erb to link to documents within the context of their exhibit%>
+<%# Note that file is now catalog/_index_header.html.erb %>
```

Comment:
"That file" being the one in Spotlight?

Reply:
No, the one in blacklight. there's not one in spotlight. I updated the comment is it better?